gc/scheduler/scheduler.go

```diff
@@ -80,17 +80,23 @@ type config struct {
 	StartupDelay duration `toml:"startup_delay"`
 }
 
-type duration time.Duration
+type duration struct {
+	time.Duration
+}
```

Comment:
Why was this change necessary? Can't you just add `MarshalText` ?

Reply:
>Why was this change necessary?

It can avoid type conversion like `time.Duration(cfg.ScheduleDelay)`.

>Can't you just add `MarshalText` ?

`UnmarshalTest` is not added by me. 
Isn't it used when loading config file? I will verify it.

Reply:
@dnephin We need `UnmarshalText` when loading config file, otherwise containerd service can not start with the error log:
```
containerd: toml: type mismatch for scheduler.duration: expected table but found string
```

Reply:
`return nil` is probably ignoring the problem, it suspect it will serialize to nothing.

My comment wasn't about `UnarmshalText`. I think it was correct and doesn't need to change.

I was suggesting it would be better to leave the type as `type duration time.Duration` and remove the other changes from this PR. I don't think the type conversion is a problem. The only thing that needs to change is adding `MarshalText`

Reply:
@dnephin Thanks for review, have address your comment.
>return nil is probably ignoring the problem, it suspect it will serialize to nothing.

I have tested, `duration` can be parsed normally regardless of the contents of `UnarmshalText`.